src/main/scala/mesosphere/marathon/core/volume/providers/DVDIProvider.scala

```diff
@@ -0,0 +1,243 @@
+package mesosphere.marathon.core.volume.providers
+
+import com.wix.accord.{ Validator, _ }
+import com.wix.accord.dsl._
+import mesosphere.marathon.core.volume.providers.DVDIProvider._
+import mesosphere.marathon.core.volume.providers.OptionSupport._
+import mesosphere.marathon.state._
+import org.apache.mesos.Protos.Volume.Mode
+import org.apache.mesos.Protos.{ Volume => MesosVolume, CommandInfo, ContainerInfo, Environment }
```

Comment:
This is also fine but we started the convention of 

`import org.apache.mesos.{Protos => MesosProtos}` 

elsewhere. Then use `MesosProtos.Volume`, `MesosProtos.CommandInfo` etc to show clearly what is from the Mesos protos.